paddle/fluid/inference/tests/api/tester_helper.h

```diff
@@ -443,15 +443,21 @@ void TestPrediction(const PaddlePredictor::Config *config,
   }
 }
 
-void SummarizeAccuracy(float avg_acc1_fp32, float avg_acc1_int8) {
+void SummarizeAccuracy(float avg_acc_fp32, float avg_acc_int8,
+                       int compared_idx) {
+  PADDLE_ENFORCE(compared_idx <= 2 && compared_idx >= 1,
+                 "Compare either top1 accuracy either mAP(top5), the "
```

Comment:
"Compare either top1 accuracy or mAP(top5), the "

Reply:
PADDLE_ENFORCE_LE(compared_idx, 2, "xxx")
PADDLE_ENFORCE_GE(compared_idx,  1, "xxx")